### PR TITLE
refactor: replace PR with Pull Requests

### PR DESCRIPTION
**Pull request by ferferga**
People not-so well versed with GH might not know what a PR is.

I didn't thought about this in code review, sorry!
Feel free to close it though, since it can also be argued that if you tinker between PRs you understand what "PR" is.

### CODE REVIEW

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]
Capitalization should just be "pull request"

**Comment by ferferga** [comment]
Really good point @nielsvanvelzen, thank you very much! 🚀🚀🚀

File: docs/general/testing/web/index.md [hunk 1]
```diff
@@ -7,16 +7,16 @@ title: Testing Jellyfin Web Clients
 
 Weekly unstable builds ship Jellyfin Server and Jellyfin Web in a complete package. Please refer to [Testing Jellyfin Server](/docs/general/testing/server/) for more info.
 
-## Testing PRs from Cloudflare Pages Deployments
+## Testing pull requests from Cloudflare Pages Deployments
```
Comment (felix920506) [inline docs/general/testing/web/index.md:10]:
```suggestion
## Testing Pull Requests from Cloudflare Pages Deployments
```
Titles use different capitalization rules than the body

File: docs/general/testing/web/index.md [hunk 2]
```diff
@@ -7,16 +7,16 @@ title: Testing Jellyfin Web Clients
 
 Weekly unstable builds ship Jellyfin Server and Jellyfin Web in a complete package. Please refer to [Testing Jellyfin Server](/docs/general/testing/server/) for more info.
 
-## Testing PRs from Cloudflare Pages Deployments
+## Testing Pull Requests from Cloudflare Pages Deployments
```
Comment (nielsvanvelzen) [inline docs/general/testing/web/index.md:10]: Personally I would prefer lowercase here too

```suggestion
## Testing pull requests from Cloudflare Pages Deployments
```

**Review by felix920506 — APPROVED** [review]

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**